src/main/java/com/corpo/coliseum/domain/service/impl/CorporationServiceImpl.java

```diff
@@ -30,18 +31,18 @@
 
     @Override
     @Transactional
-    public void create(String name, String sport) {
+    public void create(CorporationInput corporationInput) {
         Corporation.builder()
-                .name(name)
-                .sport(sport)
+                .name(corporationInput.name)
+                .sport(corporationInput.sport)
                 .build()
                 .persist();
```

Comment:
Change corporation class with CorporationInput

Reply:
Not sure this is the best thing because Corporation is Entity. It can be  confusing